pyproject.toml

```
@@ -0,0 +1,63 @@
+[tool.poetry]
+name = "saleor-app-framework-python"
+version = "0.1.0"
+description = "Saleor app framework"
+authors = ["Saleor Commerce hello@saleor.io"]
```

Maybe we should use the same line as in Saleor repo?  
```suggestion
authors = [ "Saleor Commerce <hello@saleor.io>" ]
```

---

@fowczarek  IMO This should be done when the onwer of saleor repo will be `saleor` instead of `mirumee`